tests/integration/test_dummy_integration.py

```diff
@@ -21,24 +19,25 @@
         )
 
 @pytest.fixture(scope="module")
-def test_dummy_run():
+def dummy_run():
     config_path = PROTEUS_ROOT / 'input' / 'dummy.toml'
-    ref_output = PROTEUS_ROOT / 'tests' / 'data' / 'integration' / 'dummy'
 
     runner = Proteus(config_path=config_path)
 
     runner.start()
 
-    output = Path(runner.directories['output'])
+def test_dummy_run(dummy_run):
+    out_dir = PROTEUS_ROOT / 'output' / 'dummy'
+    ref_dir = PROTEUS_ROOT / 'tests' / 'data' / 'integration' / 'dummy'
 
-    hf_all = ReadHelpfileFromCSV(output)
-    hf_all_ref = ReadHelpfileFromCSV(ref_output)
+    hf_all = ReadHelpfileFromCSV(out_dir)
+    hf_all_ref = ReadHelpfileFromCSV(ref_dir)
 
-    assert_frame_equal(hf_all, hf_all_ref, rtol=1e-4)
+    assert_frame_equal(hf_all, hf_all_ref, rtol=5e-3)
 
 @pytest.mark.xfail
```

Comment:
Maybe not part of this PR, but consider to only xfail on the assertion, not on any regular failures in the workflow itself

```suggestion
@pytest.mark.xfail(raises=AssertionError)
```